src/content/docs/Compliance/logging-monitoring-policy.md

```diff
@@ -0,0 +1,117 @@
+---
+title: Logging & Monitoring Policy
+description: Code Town Logging & Monitoring Policy
+template: doc
+---
+
+## Purpose and Scope
+
+This policy establishes requirements for logging and monitoring of Code Town's
+information systems to detect security events, maintain operational visibility,
+and maintain compliance with regulatory requirements.
+
+This policy applies to all information systems, networks, and applications owned
+or operated by Code Town, as well as all employees, contractors, and third
+parties with access to these systems.
+
+## Policy Statements: Our Commitments
+
+The following policy statements outline our core commitments and practices for
+logging and monitoring. They cover everything from log collection and retention
+to analysis and incident detection.
+
+### Logging and Monitoring Framework
+
+Code Town will implement a centralized logging and monitoring framework that
+captures and analyzes system activities across all critical IT assets. The
+framework includes:
+
+- **Log collection**: Gathering logs from servers, applications, network
+  devices, and cloud platforms.
+- **Log analysis**: Regularly reviewing and analyzing logs to detect anomalous
+  or suspicious activities.
+- **Alerting**: Configuring automated alerts for specific triggers, such as
+  unauthorized access attempts or system failures.
+- **Retention**: Storing logs securely and retaining them for a specified period
+  based on regulatory, legal, and business requirements.
+
+### Log Types and Sources
+
+The following log types are collected and monitored to ensure comprehensive
+coverage of system activities:
+
+- **Security logs**: Capturing login attempts, failed authentication, and system
+  access violations.
+- **Application logs**: Monitoring events related to the functioning and
+  performance of applications, including errors and abnormal behavior.
+- **System logs**: Tracking operating system events such as boot sequences,
+  shutdowns, and system errors.
+- **Network logs**: Capturing network traffic events, including firewall,
+  router, and switch logs.
+- **Audit logs**: Documenting user activities and system changes for compliance
```

Comment:
This is something that we still need to formalize.

Reply:
Was wondering if the Audit logs section that GCP already satisfies that or not. Assuming you mean that the "user activities" stuff is not tracked by us but I'm assuming its tracked by GCP somewhere.

Reply:
We have audit logs from our providers, but don't yet offer audit logs for shorebird-internal events (e.g. "who pressed the button to move the patch to live?") but intend to.

Reply:
In order to get this moving and reviewed I'm going to remove this line for now and will file a ticket for it on our backend for future tracking.

Reply:
Filed https://github.com/shorebirdtech/_shorebird/issues/1140 for this.